network/p2p/libp2pNode_test.go

```diff
@@ -641,21 +645,34 @@ func (suite *LibP2PNodeTestSuite) StopNode(node *Node) {
 	<-done
 }
 
-// generateNetworkingKey generates a ECDSA key pair using the given seed
-func generateNetworkingKey(seed string) (crypto.PrivKey, error) {
-	seedB := make([]byte, 100)
-	copy(seedB, seed)
-	var r io.Reader = bytes.NewReader(seedB)
-	prvKey, _, err := crypto.GenerateKeyPairWithReader(crypto.ECDSA, 0, r)
-	return prvKey, err
+// generateNetworkingKey is a test helper that generates a ECDSA flow key pair.
+func generateNetworkingKey(t *testing.T) fcrypto.PrivateKey {
```

Comment:
👍 for moving the error checking into these helpers

Reply:
thanks